src/models/image-map.ts

```diff
@@ -397,7 +397,7 @@ export const externalUrlImagesHandler: IImageHandler = {
     // 4. the data uri is "fetched" to turn it into a blob and then blob url
     // 5. the resulting imageData value will be the blob url
     // 6. the displayUrl is set to this blob url
-    if (db?.stores.user.id) {
+    if (db?.stores?.user.id) {
```

Comment:
@scytacki Does this seem like a reasonable fix? Or is the fact that `db.stores` is undefined in the context of the CMS more of a concern?

Reply:
This seems reasonable to me. The fact that the images are not uploaded when authoring is good thing. 
Please add a comment here so we'll know that in the CMS the images are not uploaded.

Reply:
@emcelroy yes i think that is right: only pasting in image URLs.